ext/opentelemetry-ext-cloud-trace/README.rst

```diff
@@ -0,0 +1,43 @@
+OpenTelemetry Cloud Trace Exporters
```

Comment:
can we rename this package to "opentelemetry-exporter-cloud-trace"? Trying to do work to normalize instrumentation names across languages, and one thing we should do is state "exporter" vs "ext" to disambiguate between an instrumentation and an exporter: https://github.com/open-telemetry/opentelemetry-specification/pull/539

Reply:
It should probably be called `opentelemetry-cloud-trace-exporter`, to keep naming somewhat similar across languages: https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/master/packages/opentelemetry-cloud-trace-exporter/package.json#L2